x-pack/plugins/secops/public/components/formatted_date/index.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment-timezone';
+import * as React from 'react';
+import { pure } from 'recompose';
+
+import { AppFrameworkAdapter } from '../../lib/lib';
+
+export const KibanaConfigContext = React.createContext<Partial<AppFrameworkAdapter>>({});
+
+export const PreferenceFormattedDate = pure<{ value: Date | string }>(({ value }) => (
+  <KibanaConfigContext.Consumer>
+    {(config: Partial<AppFrameworkAdapter>) =>
+      config && config.dateFormatTz && config.dateFormatTz
```

Comment:
is there a duplicate check here?

Reply:
Yeah, that first one should just be `config.dateFormat`. Nice catch -- thanks!